### PR TITLE
update iluvatar cpm config

### PR DESCRIPTION
**Pull request by forestlee95**
1.update iluvatar cpm config.
2.update iluvatar sdk info.
[cpm.zip](https://github.com/FlagOpen/FlagPerf/files/12426376/cpm.zip)

### CODE REVIEW

File: training/iluvatar/cpm-pytorch/README.md [hunk 1]
```diff
@@ -28,24 +13,37 @@
...
+| 训练结果       | acc,见“性能指标”               | 分类准确率(mlm_accuracy)                    |
+| 额外修改项     | 无                             |                                             |
+
+* 性能指标
+
+| 配置                | precision | fix_hp           | e2e_time | p_whole | p_train | p_core | acc   | mem       |
+| ------------------- | --------- | ---------------- | -------- | ------- | ------- | ------ | ----- | --------- |
+| BI-V100单机8卡（1x8）  | fp16      | /                |      |      |      |    | 0.92  | 10.7/32.0 |
+| BI-V100单机8卡（1x8）  | fp16      | bs=128,lr=0.002  |      |      |     |    | 0.92 |  20.3/32.0|
+| BI-V100单机单卡（1x1） | fp16      | bs=128,lr=0.0005 |          |     |    |   |       |  |
```
Comment (yuzhou03) [inline training/iluvatar/cpm-pytorch/README.md:48]: 1x1, 2x8的mem需要填写

File: training/iluvatar/cpm-pytorch/README.md [hunk 2]
```diff
@@ -28,24 +13,37 @@
...
+| 训练结果       | acc,见“性能指标”               | 分类准确率(mlm_accuracy)                    |
+| 额外修改项     | 无                             |                                             |
+
+* 性能指标
+
+| 配置                | precision | fix_hp           | e2e_time | p_whole | p_train | p_core | acc   | mem       |
+| ------------------- | --------- | ---------------- | -------- | ------- | ------- | ------ | ----- | --------- |
+| BI-V100单机8卡（1x8）  | fp16      | /                |      |      |      |    | 0.92  | 10.7/32.0 |
+| BI-V100单机8卡（1x8）  | fp16      | bs=128,lr=0.002  |      |      |     |    | 0.92 |  20.3/32.0|
+| BI-V100单机单卡（1x1） | fp16      | bs=128,lr=0.0005 |          |     |    |   |       | 10.5/32.0 |
```
Comment (yuzhou03) [inline training/iluvatar/cpm-pytorch/README.md:48]: 1x1的log跑的是bs=32, lr=0.0005

File: training/iluvatar/cpm-pytorch/README.md (hunk 1 above)
Comment (forestlee95) [reply training/iluvatar/cpm-pytorch/README.md]: 好的，正在更新数据信息。

File: training/iluvatar/cpm-pytorch/README.md (hunk 2 above)
Comment (forestlee95) [reply training/iluvatar/cpm-pytorch/README.md:48]: 已更新。

Comment (forestlee95) [reply training/iluvatar/cpm-pytorch/README.md]: [cpm_BI_V100_1x1_performance.zip](https://github.com/FlagOpen/FlagPerf/files/12435056/cpm_BI_V100_1x1_performance.zip)

**Review by shh2000 — APPROVED** [review]

**Review by yuzhou03 — APPROVED** [review]

**Pull request merged**